custom_components/tuya_local/devices/rainpoint_water_timer.yaml

```diff
@@ -0,0 +1,116 @@
+name: RainPoint Smart Water Timer
```

Comment:
This top level name should be without branding (and following HA naming convention, see below).

Just "Water timer" should be enough.